simulation/traffic_simulator/src/entity/entity_base.cpp

```diff
@@ -133,7 +134,8 @@ auto EntityBase::getVehicleCommand() const -> std::tuple<
   autoware_auto_control_msgs::msg::AckermannControlCommand,
   autoware_auto_vehicle_msgs::msg::GearCommand>
 {
-  THROW_SIMULATION_ERROR("get vehicle command does not support in ", type, " entity type");
+  THROW_SIMULATION_ERROR(
+    "get vehicle command does not support in ", getEntityTypename(), " entity type");
```

Comment:
```
"`getVehicleCommand` is not provided for ", getEntityTypename(), " type entity."
```

Reply:
modified in https://github.com/tier4/scenario_simulator_v2/pull/726/commits/0b70e1db112575734550b61484c7580437177566